packages/contracts/src/networks.ts

```diff
@@ -63,8 +63,8 @@ export const SPHINX_LOCAL_NETWORKS: Array<SupportedLocalNetwork> = [
 ]
 
```

Comment:
Technically out of scope of this PR, but does it matter that Rari is an Arbitrum L3, not an L2? ([Source](https://rarichain.org/))

I'm not sure if this impacts the cost estimate

Reply:
I don't think that matters really, I'm going to leave it out until we have a reason why we need that info.